example/speech_command/sc/evaluator.py

```diff
@@ -65,9 +65,9 @@ def __init__(self) -> None:
 
     @torch.no_grad()
     def ppl(self, audio: Audio, **kw):
-        _tensor = self._pre(audio)
+        _tensor = self._pre([audio])
```

Comment:
If the users don't config batch size, the example code keeps the original structure?

Reply:
the code need changed from 'def ppl(self, audio: Audio, **kw):' -> 'def ppl(self, audios: List[Audio], **kw):'

Reply:
the starwhale SDK did not define the precise ppl argument type requirements, so the users can customise by themselves.
https://github.com/star-whale/starwhale/blob/main/client/starwhale/api/_impl/model.py#L180

Reply:
‘If the users don't config batch size, the example code keeps the original structure?’ yes, if don't config batch size there is no need to change the code, I just changed this demo to the implementation of the default batch

Reply:
done